modules/logging-release-notes-5-9-8.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// observability/logging/logging_release_notes/logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-8_{context}"]
+= Logging 5.9.8
+This release includes link:https://access.redhat.com/errata/RHSA-2024:8315[OpenShift Logging Bug Fix Release 5.9.8].
+
+// 4.13, 4.14, 4.15, 4.16
+
+[id="openshift-logging-5-9-8-bug-fixes_{context}"]
+== Bug fixes
+// no-rn * (link:https://issues.redhat.com/browse/LOG-6169[LOG-6169])
+// no-rn * (link:https://issues.redhat.com/browse/LOG-6159[LOG-6159])
+* Before this update, the Loki Operator failed to add the default `namespace` label to all `AlertingRule` resources, which caused the User-Workload-Monitoring Alertmanager to skip routing these alerts. This update adds the rule namespace as a label to all alerting and recording rules, resolving the issue and restoring proper alert routing in Alertmanager.
+(link:https://issues.redhat.com/browse/LOG-6181[LOG-6181])
+
+* Before this update, the LokiStack ruler component view did not initialize properly, causing an invalid field error when the ruler component was disabled. This update ensures that the component view initializes with an empty value, resolving the issue.
+(link:https://issues.redhat.com/browse/LOG-6183[LOG-6183])
+
+* Before this update, an LF character in the `vector.toml` file under the ES authentication configuration caused the collector pods to crash. This update removes the newline characters from the username and password fields, resolving the issue.
+(link:https://issues.redhat.com/browse/LOG-6206[LOG-6206])
+
+[id="openshift-logging-5-9-8-CVEs_{context}"]
+== CVEs
```

Comment:
The Errata contains more CVE's

Reply:
Not from what I see? 

![image](https://github.com/user-attachments/assets/f0fb88e4-16ca-4dfb-bdc6-e3f34021cb66)